concepticondata/conceptlists/Oskolskaya-2021-254.tsv

```diff
@@ -0,0 +1,255 @@
+Oskolskaya-2021-254-167	167	look (v.)	1819	LOOK
+Oskolskaya-2021-254-168	168	nasal mucus (n.)	208	NASAL MUCUS (SNOT)
+Oskolskaya-2021-254-169	169	that	78	THAT
+Oskolskaya-2021-254-170	170	cut (v.)	1432	CUT
+Oskolskaya-2021-254-171	171	mother (n.)	1216	MOTHER
+Oskolskaya-2021-254-172	172	scratch (v.)	1436	SCRATCH
+Oskolskaya-2021-254-173	173	sun (n.)	1343	SUN
+Oskolskaya-2021-254-174	174	look for (v.)	1468	LOOK FOR
+Oskolskaya-2021-254-175	175	brain (n.)	1583	BRAIN
+Oskolskaya-2021-254-176	176	warm	1232	WARM
+Oskolskaya-2021-254-177	177	cover (v.)	761	COVER
+Oskolskaya-2021-254-178	178	woman (n.)	962	WOMAN
+Oskolskaya-2021-254-179	179	deep	1593	DEEP
+Oskolskaya-2021-254-180	180	above	1741	ABOVE
+Oskolskaya-2021-254-181	181	female (of an animal) (n.)		
+Oskolskaya-2021-254-182	182	put on (v.)	158	PUT ON
+Oskolskaya-2021-254-183	183	other	197	OTHER
+Oskolskaya-2021-254-184	184	forehead (n.)	123	FOREHEAD
+Oskolskaya-2021-254-185	185	left	244	LEFT
+Oskolskaya-2021-254-186	186	rise (v.)	568	RISE (MOVE UPWARDS)
+Oskolskaya-2021-254-187	187	dry	1398	DRY
+Oskolskaya-2021-254-188	188	how?	1239	HOW
+Oskolskaya-2021-254-189	189	break (v.)	2558	BREAK (CLEAVE)
+Oskolskaya-2021-254-190	190	where?	1237	WHERE
+Oskolskaya-2021-254-191	191	spin (v.)	121	SPIN
+Oskolskaya-2021-254-192	192	ripe (v.)	178	RIPE
+Oskolskaya-2021-254-193	193	lick (v.)	319	LICK
+Oskolskaya-2021-254-194	194	open (v.)	1156	OPEN
+Oskolskaya-2021-254-195	195	tall	711	TALL
+Oskolskaya-2021-254-196	196	bad	1292	BAD
+Oskolskaya-2021-254-197	197	bark (n.)	1204	BARK
+Oskolskaya-2021-254-198	198	breathe (v.)	1407	BREATHE
+Oskolskaya-2021-254-199	199	count (v.)	1420	COUNT
+Oskolskaya-2021-254-200	200	die (v.)	1494	DIE
+Oskolskaya-2021-254-201	201	dirty	1230	DIRTY
+Oskolskaya-2021-254-202	202	dust	2	DUST
+Oskolskaya-2021-254-203	203	fat (n.)	323	FAT (ORGANIC SUBSTANCE)
+Oskolskaya-2021-254-204	204	father (n.)	1217	FATHER
+Oskolskaya-2021-254-205	205	fear (v.)	1419	FEAR (BE AFRAID)
+Oskolskaya-2021-254-206	206	fight (v.)	1423	FIGHT
+Oskolskaya-2021-254-207	207	five	493	FIVE
+Oskolskaya-2021-254-208	208	flower (n.)	239	FLOWER
+Oskolskaya-2021-254-209	209	fog (n.)	249	FOG
+Oskolskaya-2021-254-210	210	freeze (v.)	1431	FREEZE
+Oskolskaya-2021-254-211	211	fruit (n.)	1507	FRUIT
+Oskolskaya-2021-254-212	212	green	1425	GREEN
+Oskolskaya-2021-254-213	213	guts (n.)	1334	GUTS
+Oskolskaya-2021-254-214	214	heart (n.)	1223	HEART
+Oskolskaya-2021-254-215	215	here	136	HERE
+Oskolskaya-2021-254-216	216	hunt (v.)	1435	HUNT
+Oskolskaya-2021-254-217	217	ice (n.)	617	ICE
+Oskolskaya-2021-254-218	218	lake (n.)	624	LAKE
+Oskolskaya-2021-254-219	219	live (v.)	1422	BE ALIVE
+Oskolskaya-2021-254-220	220	moon (n.)	1313	MOON
+Oskolskaya-2021-254-221	221	narrow	1267	NARROW
+Oskolskaya-2021-254-222	222	near	1942	NEAR
+Oskolskaya-2021-254-223	223	person (n.)	683	PERSON
+Oskolskaya-2021-254-224	224	push (v.)	1452	PUSH
+Oskolskaya-2021-254-225	225	river (n.)	666	RIVER
+Oskolskaya-2021-254-226	226	to be(come) rotten (v.)	1728	ROTTEN
```

Comment:
Seems as if the gloss is the state and the process at the same time?

Reply:
It's mapped to ROTTEN in Savelyev-2020-254, but I agree that the gloss is ambiguous.

Reply:
agree about the ambiguity here, but again this follows S2020.